### PR TITLE
[ICONS] warn about deprecated icons in angular dev mode

### PR DESCRIPTION
**Pull request by Shijir**
Signed-off-by: Shijir Tsogoo <stsogoo@vmware.com>

### CODE REVIEW

File: src/clr-angular/icon/icon.module.ts [hunk 1]
```diff
@@ -5,13 +5,27 @@
...
+import {isDevMode, NgModule, Type} from "@angular/core";
+
 import {ClrIconCustomTag} from "./icon";
 
 export const CLR_ICON_DIRECTIVES: Type<any>[] = [ClrIconCustomTag];
 
+// TODO: remove this warning in v0.13
+let warningThrownOnce = false;
+const DEPRECATED_SHAPE_WARNING =
+    `Starting in Clarity Icons v0.12, some icons are deprecated in their respective sets and moved to other sets. \n` +
```
Comment (gnomeontherun) [inline src/clr-angular/icon/icon.module.ts]: No need to use concatenation with multiline string, nor the `\n` character, just make a newline.

I'd do something a little more like first and last lines being something like:

```
*********
```

**Comment by Shijir** [comment]
screenshot 2018-05-22 16 12 36
Updated warning message.

File: src/clr-angular/icon/icon.spec.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+import {TestBed} from "@angular/core/testing";
+import {ClrIconModule, DEPRECATED_SHAPE_WARNING} from "./index";
+
+describe("ClrIconCustomTag", function() {
+    beforeEach(function() {
+        spyOn(console, "warn");
```
Comment (youdz) [inline src/clr-angular/icon/icon.spec.ts]: I'm not sure why this is in a `beforeEach()` and not just in the `it()`

**Comment by Shijir** [comment]
Closing this as I will submit a different solution.

**Pull request closed without merging**